index.html

```diff
@@ -4,42 +4,175 @@
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="css/style.css">
     <script src="https://use.fontawesome.com/bba5c6ae87.js"></script>
     <script src="https://use.fontawesome.com/83ac61cd0f.js"></script>
     <title>My Portfolio</title>
 </head>
 <body>
-<header class="wrapper">
-    <nav class="flex-nav">
-        <ul>
-            <li ><a class="logo" href="#">Chidiebere</a> </li>
-            <li class="menu-icon toggle-icon"><i class="fa fa-bars" aria-hidden="true"></i></li>
-        </ul>
+<header class="wrapper flex-nav">
+    <a class="logo item" href="#">Chidiebere</a>
+    <nav class="menu-icon toggle-icon item" >
+        <i class="fa fa-bars" aria-hidden="true"></i>
     </nav>
 </header>
 
 
-<section>
+<section class="item-1">
     <div class="headline">
 
         <h1 class="intro name">Hey There. I'm Chidiebere Ojingwa</h1>
-        <p class="intro profession">I'am a Software Developer</p>
+        <h2 class="intro profession">I'am a Software Developer</h2>
         <p class="intro task"> I can help you build a product, feature or <br> website
-        look through some of my work <br>and experience! If you like what you see <br> and
-        have a project you need, <br> don't hesitate to contact me</p>
+            look through some of my work <br>and experience! If you like what you see <br> and
+            have a project you need, <br> don't hesitate to contact me</p>
 
         <ul class="social">
             <li><a href="#" class="fa fa-github"></a></li>
             <li><a href="#" class="fa fa-linkedin"></a></li>
             <li><a href="#" class="fa fa-angellist"></a></li>
             <li><a href="#" class="fa fa-twitter"></a></li>
-            <li> <a href="#" class="fa fa-medium"></a></li>
+            <li><a href="#" class="fa fa-medium"></a></li>
+        </ul>
+    </div>
+</section>
+
+<section class="item-2">
+    <!-- <div class="recentWork"> -->
+        <h2 class="recentWork">My Recent Works</h2>
+    <!-- </div> -->
+    <img class="line" src="assets/line.png" alt="straight line">
+    <div class="works first">
+        <img class="firstProjectImg" src="./assets/imgplaceholder.png" alt="project Image">
+        <h2 class="multi"> Multi-Post Stories</h2>
+        <p class="story">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+        <ul class="grid-container">
+            <li class="stack">css</li>
+            <li class="stack">html</li>
+            <li class="stack">bootstrap</li>
+            <li class="stack">Ruby</li>
         </ul>
+        <a class="project">See Project</a>
```

Comment:
- [ ] As requested by the former code reviewer, which is also part of the project requirements, and a best practice to stick to the client's design. Please make sure you buttons("See Project") matches those in the design. 

***The design***
![chiD](https://user-images.githubusercontent.com/65852621/134683256-36376438-9264-4c86-9f7e-9f867a0ef415.png)

***Your implementation***
![chiI](https://user-images.githubusercontent.com/65852621/134682970-b5431fdb-4f5e-4776-b1c0-2fb4b417798d.png)